src/GslCore/DnaCreation.fs

```diff
@@ -49,13 +50,18 @@ let adjustToPhysical (feat:sgd.Feature) (f:RelPos) =
 /// these are relative to the gene, not the genome for now.  We transform
 /// to genomic coordinates below.
 /// Transform all non gXXX forms of gene into gXX forms.
-let translateGenePrefix (gd : GenomeDef) (gPart : StandardSlice) =
+let translateGenePrefix (pragmas:PragmaCollection) (gd : GenomeDef) (gPart : StandardSlice) =
     match gPart with
-    | PROMOTER ->
-        {left = {x = -500<OneOffset>; relTo = FivePrime};
+    | PROMOTER ->   
+        { left = { x = match pragmas.TryFind "promlen"  with

```

Comment:
Please normalize the indentation in this record expression.

Reply:
It might be more readable to move the match expression out of the record expression and bind it to a local.

Reply:
PragmaCollection already has a helper for this most common case; match on `pragmas.TryGetOne "promlen"` which already unpacks the first argument.

Reply:
These comments apply to the two similar changes in this module.